src/nextcloud/talk/recording/Participant.py

```diff
@@ -464,6 +473,7 @@ def __init__(self, browser, nextcloudUrl, width, height, env, parentLogger):
         :param height: the height of the browser window.
         :param env: the environment variables, including the display to start
                     the browser in.
+        :param executable_path: the direct path to the browser executable
```

Comment:
_executable_path_ is not the path to the browser executable, but to the Selenium driver executable